_data/lifecycle-phases.yml

```
@@ -0,0 +1,161 @@
+          Don't add comments in the <code>uswds-proposals</code> repo.
+          </p><p>
+          Find all components with proposals in progress on the discussion board.
+      - name: Proposal open for comment
+        description: |
+          The community has a minimum of 45 days to comment on the complete proposal.
+        starts_when: The USWDS core team shares the complete proposal with the community for final feedback.
+        ends_when: The comment period closes and the core team starts evaluating feedback.
+        contribute_by:
+          - Sharing your thoughts about the proposal
+          - Asking questions about the proposal
+          - Voting for the related component discussion
+        find_more: |
+          <b>Note:</b> Keep all questions and comments related to the the proposal in the public discussion.
+          Don't add comments in the <code>uswds-proposals</code> repo.
+          </p><p>
+          Find all components with proposals open for comment on the discussion board.
+      - name: Proposal evaluation
+        description: |
+          The USWDS core team evaluates the proposal for inclusion in the design system.
+          After evaluation, the core team will approve the proposal, return it for revision, or mark as “will not pursue.”
+        starts_when: The USWDS core team starts evaluating the proposal.
+        ends_when: The USWDS core team shares its decision with the community.
+        find_more: Find the components with proposals under evaluation on the discussion board.
+      - name: Approved
+        description: |
+          The USWDS core team determined that the component could be a good fit for the design system.
+          Components in this phase are ready for development but have not yet been assigned.
+        starts_when: The USWDS core team approves the component proposal.
+        ends_when: The component is assigned for development.
+        contribute_by:
+          - Volunteering to take the assignment and develop the component
+        find_more: Find all approved components on the discussion board.
+        reach_out: <a href="mailto:uswds@gsa.gov">Reach out to the USWDS core team</a> if you are interested in building an approved component.
+      - name: Conditionally approved
+        description: |
+          The USWDS core team determined that the component could be a good fit for the design system, but other work must be completed before development can be scheduled.
+          Development for these components will be put on hold until the related work is finished.
+          Once the related work is complete, the component will move to the “approved” state.
+        starts_when: The USWDS core team approves the component proposal.
+        ends_when: The dependent work is complete and the component moves to approved status.
+        find_more: Find all conditionally approved components on the discussion board.
```

Need to add link once it is available.
```suggestion
        find_more: Find all [conditionally approved components]() on the discussion board.
```